src/panels/SkyBrowserPanel/BrowserTabs/Settings.tsx

```diff
@@ -24,31 +23,18 @@ export function Settings({ id }: Props) {
     luaApi?.skybrowser.setBorderColor(selectedBrowserId, r, g, b);
   }
 
-  function deleteBrowser() {
-    if (!id) {
-      return;
-    }
-    // If there are more browsers, select another browser
-    if (browserIds.length > 1) {
-      const otherBrowsers = browserIds.filter((b) => b !== id);
-      luaApi?.skybrowser.setSelectedBrowser(otherBrowsers[0]);
-    }
-    luaApi?.skybrowser.removeTargetBrowserPair(id);
-  }
-
   return (
     <Stack gap={5} my={'lg'}>
       <Title order={2}>Settings</Title>
+      {/* Just using this variable here so the linter is happy */}
```

Comment:
Why is this comment needed? :) I would suggest to remove it

Reply:
It's to make it easier to merge when i merge the skybrowser settings pr

Reply:
Aha, so the idea is to remove it on merge of that one?